Snapyr/Classes/SnapyrSDK.h

```diff
@@ -108,9 +108,18 @@ NS_SWIFT_NAME(Snapyr)
 - (void)track:(NSString *)event;
 
 - (void)setPushNotificationToken:(NSString*)token;
-- (void)pushNotificationReceived:(SERIALIZABLE_DICT _Nullable)info;
-- (void)pushNotificationTapped:(SERIALIZABLE_DICT _Nullable)info;
-- (void)pushNotificationTapped:(SERIALIZABLE_DICT _Nullable)info actionId:(NSString* _Nullable)actionId;
+- (void)setPushNotificationTokenData:(NSData*)tokenData;
+
+- (void)pushNotificationReceivedWithResponse:(UNNotificationResponse *)response;
+- (void)pushNotificationReceived:(UNNotification *)notification;
```

Comment:
@igor-snapyr just one note on this one: let's try to keep the method signatures backwards compatible with the previous version, so that people who are already using a previous SDK version can drop in the update without any issues.

(too bad we can't go with the version that's just more sensible now! 😞 )

Otherwise lgtm!